app/subscriber/public/keycloak.json

```diff
@@ -1,6 +1,6 @@
 {
   "realm": "tno",
-  "auth-server-url": "http://localhost:40001/auth/",
+  "auth-server-url": "http://localhost:50001/auth/",
```

Comment:
Ha, we'll need to do a lot more work than this to get the subscriber app working.

Reply:
Oops! That was for my local :)

Reply:
Edited the wrong file lol will remove